reference/test/solver/batch_bicgstab_kernels.cpp

```diff
@@ -247,6 +249,42 @@ TYPED_TEST(BatchBicgstab, CanSolveEllSystem)
 }
 
 
+TYPED_TEST(BatchBicgstab, CanSolveCsrSystem)
```

Comment:
Maybe this can be change to the uniform test for different matrix format.
(not need in the pr)

Reply:
Yes, maybe I will handle it in another PR